rcljava/include/org_ros2_rcljava_graph_NodeNameInfo.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <jni.h>
+/* Header for class org_ros2_rcljava_graph_NodeName */
+
+#ifndef ORG_ROS2_RCLJAVA_GRAPH_NODENAMEINFO_H_
+#define ORG_ROS2_RCLJAVA_GRAPH_NODENAMEINFO_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+/*
+ * Class:     org_ros2_rcljava_graph_NodeName
+ * Method:    nativeFromRCL
+ * Signature: (J)V
+ */
+JNIEXPORT void
+JNICALL Java_org_ros2_rcljava_graph_NodeName_nativeFromRCL(
```

Comment:
```suggestion
JNICALL Java_org_ros2_rcljava_graph_NodeNameInfo_nativeFromRCL(
```

Reply:
This code is gone, see https://github.com/osrf/ros2_java/pull/31/commits/51de623a8da281d95951795ac0af9be48fc23bf6